templates/angular-package/config/default.json

```diff
@@ -0,0 +1,4 @@
+{
+    "test-config":"Hello from config"
+  
+}
```

Comment:
Does this file need to be commited?

Reply:
I will add a default value, for the developers